.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: python
-python: "3.4"
+python: "3.5"
```

Comment:
That looks like a non production version today. That makes me a little uncomfortable.